src/extensions/resolveSource.ts

```diff
@@ -4,14 +4,25 @@ import * as semver from "semver";
 import * as api from "../api";
 import { FirebaseError } from "../error";
 import { confirmUpdateWarning } from "./updateHelper";
+import { logger } from "..";
```

Comment:
I didn't know you could import like this!

Reply:
Ahhh, you can, but we generally avoid it - I'm pretty sure autoimport chose this over ../logger. Switching this.